JSDemos/Demos/ButtonGroup/Overview/description.md

```diff
@@ -1,3 +1,17 @@
-ButtonGroup is a UI component that displays a set of toggleable buttons and allows a user to select one or several of them. This demo illustrates how to configure the ButtonGroup to enable single and multiple selection.
+ButtonGroup displays a set of toggleable buttons and allows users to select one or several of them. This demo illustrates how to configure the ButtonGroup: bind ButtonGroup to data, enable or disable multiple selection and customize the ButtonGroup appearance.
+
+## Bind ButtonGroup to Data
+
+Use the [items](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/) array to populate ButtonGroup with data. You can specify the following fields in the data objects: [hint](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/#hint), [icon](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/#icon), [text](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/#text), [type](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/#type) and others.
+
+## Process Button Selection
+
+Set the [keyExpr](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/#keyExpr) property to the data field that supplies keys used to distinguish between the selected buttons. In this demo, the [selectedItemKeys](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/#selectedItemKeys) property contains the keys of the selected buttons and allows component to select buttons initially. To enable multiple selection, set the [selectionMode](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/#selectionMode) property to `multiple`.
```

Comment:
> Set the [keyExpr](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/#keyExpr) property to the data field that supplies keys used to **distinguish** between the selected buttons.

What do you mean by "distinguish" in this sentence exactly? If you mean to literally "distinguish", then you don't need the pronoun "between" at all...

> In this demo, the [selectedItemKeys](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/#selectedItemKeys) property contains the keys of the selected buttons and allows component to select buttons initially.

The word "component" here is missing an article.

What do you mean by this? I cannot say I understand this sentence:

> to select buttons initially.

Reply:
"to select buttons initially" -> Customers can predefine buttons that would be already selected on the initialization.
What do you think about the following?

In this demo, the [selectedItemKeys](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/#selectedItemKeys) property contains the keys of the selected buttons. This property allows you to predefine selected buttons.

Reply:
Yeah, this version will definitely do. Thank you.